drivers/base/kdvm/kdvm.h

```diff
@@ -15,7 +15,7 @@
 #include <arc/arc.h>
 
 #undef RtlEqualMemory
-#define RtlEqualMemory(a, b, c) (RtlCompareMemory(a, b, c) != c)
+#define RtlEqualMemory(a, b, c) (RtlCompareMemory((a), (b), (c)) == (c))
```

Comment:
Notes, while here:
* `c` "should" be evaluated once only...
* I would suggest to s/a, b, c/s1, s2, l/ at least, to be a bit more explicit.

Reply:
> that macro is used only once in one single place, where "c" is a numerical constant.

I saw that.Then, I "wonder" why you are adding "useless" parentheses.

> In addition, if one were using RtlCompareMemory explicitly, that "c" parameter would be written twice explicitly, so there's not much gain there.

I understand but do not agree:
if one uses the function, one is responsible for writing the call and its check correctly;
if one uses the macro, one should be safe without worrying about the parameter being evaluated twice "behind the scene".